pkg/kubelet/cm/cpumanager/topology/topology_test.go

```diff
@@ -983,6 +991,42 @@ func TestCPUCoreID(t *testing.T) {
 	}
 }
 
+func TestCPUsPerCore(t *testing.T) {
```

Comment:
this will conflict with https://github.com/kubernetes/kubernetes/pull/124561 - and 124561 is more correct IMO

Reply:
resolved in 512ca4b9888c303b1b0aef5469293a099bc4cd9b